bats/tests/pyrsia_cli.bats

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+
+COMMON_SETUP='common-setup'
+
+DOCKER_COMPOSE_DIR="$REPO_DIR/bats/tests/resources/docker/docker-compose_bootstrap.yml"
+
+# Individual tests should timeout after 3 minutes
+export BATS_TEST_TIMEOUT=60
+
+setup_file() {
+  load $COMMON_SETUP
+  _common_setup_file "$DOCKER_COMPOSE_DIR"
+}
+
+teardown_file() {
+  load $COMMON_SETUP
+  _common_teardown_file
+}
+
+setup() {
+  load $COMMON_SETUP
+  _common_setup
+  PYRSIA_CLI="$PYRSIA_TARGET_DIR"/pyrsia
+}
+
+@test "Testing 'pyrsia help' CLI, check if the help is shown." {
+  # run pyrsia help
+  run "$PYRSIA_CLI" help
+  # check if pyrsia help is shown
+  assert_output --partial 'USAGE:'
+  assert_output --partial 'OPTIONS:'
+  assert_output --partial 'SUBCOMMANDS:'
+
+  # run pyrsia help
+  run "$PYRSIA_CLI" -h
+  # check if pyrsia help is shown
+  assert_output --partial 'USAGE:'
+  assert_output --partial 'OPTIONS:'
+  assert_output --partial 'SUBCOMMANDS:'
+}
+
+@test "Testing 'pyrsia ping' CLI, check if the node is up and reachable." {
+  # run pyrsia ping
+  run "$PYRSIA_CLI" ping
+  # check if pyrsia ping returns errors
+  refute_output --partial 'Error'
+}
+
+@test "Testing 'pyrsia status' CLI, check if the node is connected to peers." {
+  # run pyrsia status
+  run "$PYRSIA_CLI" status
+  # check if pyrsia node has peers, fail if doesn't or error
+  refute_output --partial '0'
+  refute_output --partial 'Error'
+}
+
+@test "Testing 'pyrsia list' CLI, check if the node returns the list of peers." {
+  skip
```

Comment:
This test can be included again, since the issue has been resolved with https://github.com/pyrsia/pyrsia/pull/1086.